addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSvnScanRule.java

```diff
@@ -698,4 +693,23 @@ private boolean findSourceCodeSVN(HttpMessage originalMessage) throws Exception
         }
         return false;
     }
+
+    private AlertBuilder buildAlert(
+            int confidence,
+            String url,
+            String attackFilename,
+            String urlfilename,
+            String evidence) {
+        return newAlert()
+                .setConfidence(confidence)
+                .setUri(url)
+                .setAttack(attackFilename)
+                .setOtherInfo(getExtraInfo(urlfilename, attackFilename))
+                .setEvidence(evidence);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(buildAlert(Alert.CONFIDENCE_MEDIUM, "", "", "", "").build());
```

Comment:
I need to figure out actual values for an example. I don't have any local or unittest sources.